lesson_01/dasiaenglish/README.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title> My README in HTML </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="heading">
+            <h4>README.md</h4>
+        </div>
+
+        <!--<div class="line"></div>-->
+        <div class="content">
+            <h1>Dasia English </h1>
+            <div class="line"></div>
+
+            </head>
+            <h2>Get to know me :)</h2>
+            <div class="line"></div>
+            <p>Hello everyone! I am excited to be in this program and further my education in tech. I graduated from the
+                University
+                of Delaware majoring in Entrepreneurship, and minored in Disability Studies. Not to mention I was on the
+                Coed
+                Cheer
+                team and named captain my Senior year. I have 19 years of experience in cheerleading. With that
+                experience,
+                I
+                also
+                have won a lot of cool things in those years. My most recent accomplishment is being a 4x Collegiate
+                Champion in
+                a
+                row.</p>
+
+            <h3>2024 Game Day Champions (6peat)</h3>
+            <div class="line"></div>
+
+            <img id="image" src="/lesson_01/dasiaenglish/images/Celebration.JPG" height="400/" style="max-width: 100%">
```

Comment:
Link to picture is broken, please fix. Also, height attribute contains an invalid character ("/").